src/scripts/clipperUI/components/modeButtonSelector.tsx

```diff
@@ -1,28 +1,39 @@
-import {ClientType} from "../../clientType";
 import {Constants} from "../../constants";
-import {Experiments} from "../../experiments";
-
 import {AugmentationHelper} from "../../contentCapture/augmentationHelper";
-
 import {ExtensionUtils} from "../../extensions/extensionUtils";
 import {InvokeMode} from "../../extensions/invokeOptions";
-
 import {Localization} from "../../localization/localization";
-
 import {ClipMode} from "../clipMode";
 import {ClipperStateProp} from "../clipperState";
 import {ComponentBase} from "../componentBase";
-
 import {ModeButton} from "./modeButton";
 
+export interface NoAriaPropsForModeButton {
+	imgSrc: string;
+	label: string;
+	myMode: ClipMode;
+	selected?: boolean;
+	tabIndex?: number;
+	onModeSelected: (modeButton: ClipMode) => void;
+	tooltipText?: string;
+}
+
+export interface AriaPropsForModeButton {
+	"aria-posinset": number;
+	"aria-setsize": number;
+}
+
+export interface ModeButtonProps extends NoAriaPropsForModeButton, AriaPropsForModeButton { }
+
 class ModeButtonSelectorClass extends ComponentBase<{}, ClipperStateProp> {
+
 	onModeSelected(newMode: ClipMode) {
 		this.props.clipperState.setState({
 			currentMode: this.props.clipperState.currentMode.set(newMode)
 		});
 	};
 
-	private getScreenReaderOnlyElementThatAnnouncesCurrentMode(currentMode: ClipMode) {
+	private getScreenReaderOnlyElementPropsThatAnnouncesCurrentMode(currentMode: ClipMode) {
```

Comment:
Standardize naming of these methods (in some of these, props is the last word, but not always)